getParams.m

```diff
@@ -64,11 +64,22 @@
 
 %% more parameters to get according to thetype of experiment
 if strcmp(expParameters.task,'tapTraining')
+    
     % get tapping training parameters
     [cfg,expParameters] = getTrainingParameters(cfg,expParameters);
 
+    if ~cfg.debug
+        % download missing audiofiles from Dropbox
+        url = 'https://www.dropbox.com/sh/20hgit6xoqxsbt0/AABnGrj6XDH08zQ5ilICLARVa?dl=1'; 
+        disp('downloading audio files from Dropbox...'); 
+        urlwrite(url,'stimuli.zip'); 
+        unzip('stimuli.zip','stimuli'); 
+        delete('stimuli.zip')
+        disp('audio downloaded successfully'); 
+    end
```

Comment:
The way this works is that it will "update" the stimuli folder. So whatever is already there will not be deleted. So my idea was only having the large datafiles on Dropbox and downloading them this way. Grahn txt files can be on github, no problem. I guess it makes things quite flexible, hopefully it's a good thing :D